src/BenchmarksDriver/Program.cs

```diff
@@ -1142,7 +1142,18 @@ public static int Main(string[] args)
                         spanLoop = spanLoop + 1;
                     } while (DateTime.UtcNow - startTime < span);
 
-                    await sqlTask;
+                    if (!sqlTask.IsCompleted)
+                    {
+                        Log("Job finished, waiting for SQL to complete.");
+                        try
+                        {
+                            await sqlTask;
+                        }
+                        catch (Exception ex)
```

Comment:
Looks too easy to just catch the exception and display it ... but I don't know better.

Reply:
Oh wait, I already catch in the `ContinueWith` I'll remove this